4-HTTP/client/utils/fetchAPI.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
```

Comment:
fetchAPI is a bad name, maybe call it `todosAPI`

Reply:
Changed it to todoAPI.